indra/newview/llinventorybridge.cpp

```diff
@@ -3680,6 +3680,11 @@ void LLFolderBridge::performAction(LLInventoryModel* model, std::string action)
 // <FS:TT> Patch: ReplaceWornItemsOnly
     else if ("replaceitems" == action)
     {
+        //<FS:AR> FIRE-31508: check folder limit
+        if (modifyOutfitExceedsWearFolderLimit())
```

Comment:
Expand up once, and you'll see modifyOutfit(bool).
In there the check/nag is done, so it seemed to be the easier thing to do: use what is already there.